### ISSUE TITLE
Xmas Event

### ISSUE THREAD

**Issue by Mortal792** [devbutlazy/Blum-AutoClicker#20]
Can you update it for the Xmas event? You can easily get 1200 BPs per game

**Comment by Mortal792** [comment]
![image](https://github.com/user-attachments/assets/2b152774-0235-4ec3-96e4-1b509fec7b4a)

**Comment by qt333** [comment]
git pull

**Comment by Mortal792** [comment]
yeah, I didn't check the pull requests before I posted this, my bad! xD
